support-frontend/assets/components/page/page.tsx

```diff
@@ -3,7 +3,8 @@ import type { ReactNode } from 'react';
 import { useEffect } from 'react';
 import CsrBanner from 'components/csr/csrBanner';
 import { SkipLink } from 'components/skipLink/skipLink';
-import { classNameWithModifiers } from 'helpers/utilities/utilities';
+import { pageDefaultStyles } from 'stylesheets/emotion/pageDefaults';
```

Comment:
I wonder if it's confusing to have this in a directory called `stylesheets`, it makes me thing it's going to contain .css (or .scss) files rather than css in js. Perhaps just `styles/pageDefaults`?